internal/sbi/nssaiavailability/routers.go

```diff
@@ -15,11 +15,15 @@ import (
 
 	"github.com/gin-gonic/gin"
 
+	nssf_context "github.com/free5gc/nssf/internal/context"
 	"github.com/free5gc/nssf/internal/logger"
+	"github.com/free5gc/nssf/internal/util"
 	"github.com/free5gc/nssf/pkg/factory"
 	logger_util "github.com/free5gc/util/logger"
 )
 
+const serviceName string = "nnssf-nssaiavailability"
```

Comment:
Use the definition here https://github.com/free5gc/openapi/blob/main/models/model_service_name.go